src/App.css

```diff
@@ -25,7 +26,9 @@ nav {
   font-size: 1.2rem;
   line-height: 1.1;
   height: 90vh;
+  padding-bottom: 0px!important;
 }
```

Comment:
Hi @Tanmayi03 it seems that you have not pulled the recent changes as `padding-right: 0px!important;` is already added in the code. Please do `git pull` and then commit your changes. Also, I am not sure if this is the correct solution as we are not using `content `anywhere for now.